project.scala

```diff
@@ -1,8 +1,8 @@
-//> using scala 3.3.0
+//> using scala 3.3.3
 
 //> using options -feature
 //> using options -deprecation
 //> using options -source:future
 //> using options -Ykind-projector:underscores
 
-//> using lib org.scalameta::munit:0.7.29
+//> using toolkit default
```

Comment:
I think we should stick with the explicit dependency on munit (but bump the version to 1.0). Pulling in the toolkit will grab too many libraries. Also, can we make the same updates to build.sbt?

Reply:
Sure!